src/database/sql/sequelize/repositories/clinical/careplan/careplan.repo.ts

```diff
@@ -408,4 +422,61 @@ export class CareplanRepo implements ICareplanRepo {
         }
     };
 
+    public updateRisk = async (model: EnrollmentDomainModel): Promise<EnrollmentDto> => {
+        try {
+            const updateRisk = await CareplanEnrollment.findOne({
+                where : {
+                    PatientUserId : model.PatientUserId,
+                    Provider      : model.Provider,
+                    PlanCode      : model.PlanCode,
+                },
+            });
+
+            if (model.Complication != null) {
+                updateRisk.Complication = model.Complication;
+            }
+            if (model.HasHighRisk != null) {
+                updateRisk.HasHighRisk = model.HasHighRisk;
+            }
+    
+            await updateRisk.save();
+
+            return EnrollmentMapper.toDto(updateRisk);
+        } catch (error) {
+            Logger.instance().log(error.message);
+            throw new ApiError(500, error.message);
+        }
+    };
+
+    public deleteFutureCareplanTask = async (enrollment: EnrollmentDomainModel): Promise<number> => {
```

Comment:
Change the name from 'deleteFutureCareplanTask' to 'deleteFutureCareplanTasks'.

Reply:
Done